### ISSUE TITLE
[New_profile] Changing the config setting for minimum age does not change the behaviour of the new profile page

### ISSUE THREAD

**Issue by KLaFleur** [aces/Loris#9097]
Fails step 9 of testing instructions: 

`Go to the Configuration module and change the "Minimum candidate age" value, ensure that "Date of Birth" and "Date of Birth Confirm" fields properly reflects the changes. If no value is entered, this two fields should have the current day as the maximum possible value to choose. [Manual Testing]`

After properly updating minimum age, there is no change in behavior of the date selector, and a candidate with an age below min can be successfully created.

**Comment by regisoc** [comment]
@KLaFleur From what I can understand of the [date calculation](https://github.com/aces/Loris/blob/9fbe7a6cf4e21d6946d79d78cdd9224a0f730678/modules/new_profile/php/new_profile.class.inc#L52), not sure to quite catch the issue. So, here is an example:

![minmaxdates](https://github.com/aces/Loris/assets/25327259/cb1add77-2464-403d-b4fc-242da45f15aa)

<details>
<summary>I tried on my vm. Creating a new user with these config, an error the expected min/max dates</summary>

![image](https://github.com/aces/Loris/assets/25327259/8b8f819b-dd01-43cb-a849-2aca9b163a22)

![image](https://github.com/aces/Loris/assets/25327259/6c70b643-1a80-4231-a814-294f95ff753a)

</details>

<details>
<summary>Date picker also disabled all year values before 1916 and after 2018, so that is good too</summary>

![image](https://github.com/aces/Loris/assets/25327259/6a1b134f-e4e6-4328-9d9f-45a460060421)

![image](https://github.com/aces/Loris/assets/25327259/19985368-277a-4b5e-8642-4e31d9e98962)

</details>

<details>
<summary>The `DateElement` min and max values for both date elements of the page, and looks good too</summary>

![image](https://github.com/aces/Loris/assets/25327259/12f8b63a-4f66-4e9f-931a-070eeea80dc7)

</details>

<details>
<summary>Changing the `Minimum candidate age` value in config to 2 also updates the date element values to the expected values</summary>

![image](https://github.com/aces/Loris/assets/25327259/36d01a4a-9afa-464f-8d7b-be52b329dc0e)

</details>

Maybe I am missing the point here. Is something wrong with the date element?

**Comment by regisoc** [comment]
Just tested on the 26.0 VM. There is a difference. Min date has effectively no limit.
Not sure why, does not happen on my vm.

![image](https://github.com/aces/Loris/assets/25327259/5a71f5c0-0e86-453f-9db0-708ffb492016)

**Comment by regisoc** [comment]
@KLaFleur can you try/reproduce it on your VM?

**Comment by KLaFleur** [comment]
Yes I'll give it a try -- found the same thing on the 26 test VM: one is able to create candidates who are born in the future!

**Comment by regisoc** [comment]
It appears correct on my VM but not on 26.0 testing VM.
Let me know how it goes on yours.

**Comment by regisoc** [comment]
@driusan can you or someone else try this on another VM? I cannot reproduce it on my VM.
@KLaFleur has to change environment on his only VM for LORIS Core which can be a pain and take some time.
While he is asking for another VM for LORIS Core stuffs, trying the case I described at the top comment could be done easily (changing 2-4 config values and checking the module GUI) by anyone with a LORIS VM with up-to-date set up.

**Comment by regisoc** [comment]
Ticket for the VM creation is on redmine. 
@KLaFleur if you have the VM ready quickly, you will be able to install LORIS Core and test on it.
Let us know here how it goes.

**Comment by KLaFleur** [comment]
can confirm this is not a problem on my vm (or the 26 test vm anymore!)
